docs/source/getting-started/quickstart.rst

```diff
@@ -127,16 +127,31 @@ This may show multiple clusters, if you have created several:
 
 SSH into clusters
 =================
-To log into a cluster, simply run :code:`ssh <cluster_name>`:
+Simply run :code:`ssh <cluster_name>` to log into a cluster:
 
 .. code-block:: console
 
   $ ssh mycluster
 
+:ref:`Multi-node clusters <dist-jobs>` work too:
+
+.. code-block:: console
+
+  # Assuming 3 nodes.
+
+  # Head node.
+  $ ssh mycluster
+
+  # Worker nodes.
+  $ ssh mycluster-worker1
+  $ ssh mycluster-worker2
+
+The above are achieved by adding appropriate entries to ``~/.ssh/config``.
```

Comment:
After reading this sentence, I'm not too clear what is being done. Either make this more specific (we create a new file and link it in `~/.ssh/config`) or dont mention it at all (Sky is magic)

Reply:
hmm, thinking more about this I think it's okay, since this is quickstart and details are not necessary. Curious folks can always `cat ~/.ssh/config` to see what's going on

Reply:
Yep, I think we shouldn't put too much detail here, but a pointer.